frontend/src/old-pages/Configure/Components.tsx

```diff
@@ -231,7 +238,7 @@ function CustomAMISettings({
       })
     }
 
-  const toggleCustomAmi = (event: any) => {
+  const toggleCustomAmi = (_e: any) => {
```

Comment:
suggestion: I think here you can remove `_e` entirely, or otherwise we should type it appropriately with `NonCancelableEventHandler<ToggleProps.ChangeDetail>` which should be exported by `@awsui`